BurnerB/FreeMentors-V2
# 168368190 Fix failing travis

#### What does this PR do?
- Fixes failing travis build

#### Description of Task to be completed
- Set the app environment variables on travis Ci
- Configure the .travis.yml file

#### How should this be manually tested?
 - Visit https://travis-ci.org/, make sure the branch  is set  to bg-fix-failing-travis-168368190.
- The build should be passing

#### What are the relevant pivotal tracker stories?
[#168368190](https://www.pivotaltracker.com/story/show/168368190)

#### Screenshots (if appropriate)
![Travis passing](https://user-images.githubusercontent.com/45785786/64537625-d3120380-d323-11e9-840e-6d37c244d432.png)
